tests/data/reference/test_monovariate_besttrack_ensemble/vortex_1_variable_perturbation_2.22

```diff
@@ -58,11 +58,11 @@ AL, 06, 2018091518,   , BEST,   0, 336N,  798W,  47,  999, TS,  34, NEQ,  136,
 AL, 06, 2018091600,   , BEST,   0, 336N,  802W,  43,  999, TS,  34, NEQ,  128,  127,  148,  152, 1013,  240, 110,  50,   0,   L,   0,    , 270,   2,   FLORENCE,  22
 AL, 06, 2018091606,   , BEST,   0, 336N,  808W,  38, 1000, TS,  34, NEQ,  126,  124,    0,    0, 1013,  260, 110,  40,   0,   L,   0,    , 270,   3,   FLORENCE,  23
 AL, 06, 2018091612,   , BEST,   0, 336N,  815W,  33, 1003, TS,  34, NEQ,    0,    0,    0,    0, 1013,  280, 140,  40,   0,   L,   0,    , 270,   3,   FLORENCE,  24
-AL, 06, 2018091618,   , BEST,   0, 341N,  821W,  28, 1007, TD,   0,    ,    0,    0,    0,    0, 1013,  300, 140,  40,   0,   L,   0,    , 315,   4,   FLORENCE,  25
```

Comment:
@WPringle thanks for checking, just to double check, can you explain why these values changes from `0` to some large values? Is the new approach avoiding some `nan` values that we previously used to get?

Reply:
@SorooshMani-NOAA Yes, the old approach didn't actually converge in some cases so would just get a 0. Now it should always converge to something if it is possible to.